R/Measure.R

```diff
@@ -141,6 +141,10 @@ getDefaultMeasure = function(x) {
     x$task.desc$type
   else if (inherits(x, "Learner"))
     x$type
+  else if (!inherits(try(checkLearner(x), silent = TRUE), "try-error"))
```

Comment:
I really dislike this line, if someone has a better solution I'm happy to hear it